Extensions/IISWebAppDeploy/Src/CHANGELOG.md

```diff
@@ -4,7 +4,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
-## [1.2.5] - 25th November 2016
+### Added
+- Webapp folder supported
```

Comment:
Support for deploying web application using build output (folder) from drop location
Support for deploying web application using build output zip